src/Playwright/Request.cs

```diff
@@ -72,7 +72,7 @@ internal Request(IChannelOwner parent, string guid, RequestInitializer initializ
 
         public async Task<IResponse> ResponseAsync() => (await _channel.GetResponseAsync().ConfigureAwait(false))?.Object;
 
-        public JsonDocument PostDataJSON(JsonDocumentOptions documentOptions = default)
```

Comment:
I consider this important to users. Without this argument, this method will be useless  to many times.

Reply:
I'm looking at the properties and they seem to be for lax parsing, which we don't need. MaxDepth - we should hard-code to -1, i'll do it.